### PR TITLE
remove hardsuits from lockers that have suit storages

### LINKED ISSUE #16431: warden has 2 hardsuits
## Description
1 in locker 1 in suit storage

i will fix in a bit if nobody else does

### PR DESCRIPTION
**Pull request by deltanedas**
## About the PR
fixes #16431
should only be merged once every map has suit storages

also gave engi locker a t-ray since:
1. atmos locker has gas analyzer
2. it had just insuls and gasmask lol

**Media**
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
no cl no fun

### CODE REVIEW

**Comment by tom-leys** [comment]
@deltanedas Where did my Atmos Hardsuit / Firesuit go? It's not really possible to do the emergency responder job without these. Am I missing something about an alternative locker?

**Comment by AJCM-git** [comment]
inside `SuitStorageAtmos`

**Comment by tom-leys** [comment]
Have all maps got these new lockers in them? Do we need to do something to bridge the game for those that don't?

File: Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml [hunk 1]
```diff
@@ -80,15 +80,10 @@
   components:
   - type: StorageFill
     contents:
-      - id: ClothingOuterHardsuitAtmos
-      - id: ClothingMaskGasAtmos
-      - id: YellowOxygenTankFilled
-      - id: NitrogenTankFilled
-      - id: ClothingOuterSuitAtmosFire
-      - id: ClothingHeadHelmetAtmosFire
```
Comment (tom-leys) [inline Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml:88]: The firesuit isn't currently in SuitStorageAtmos

```
      - id: ClothingOuterSuitAtmosFire
      - id: ClothingHeadHelmetAtmosFire
```

**Review by tom-leys — COMMENTED** [review]
You need to ensure that these lockers are in all played maps prior to merging this PR. 

I'm not very informed on where these new lockers are / so it might be my ignorance talking here.

**Comment by AJCM-git** [comment]
> Have all maps got these new lockers in them? Do we need to do something to bridge the game for those that don't?

mappers are on it, and as said in the pr body this wont be merged until every map has been updated

File: Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml (hunk 1 above)
Comment (deltanedas) [reply Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml:88]: ye it needs that

**Comment by deltanedas** [comment]
> > > > > > > > should only be merged once every map has suit storages

:trollface:

**Comment by Flareguy** [comment]
I don't think the firesuit should go in suit storage, it's usually used for spaceworthy suits in SS13. In fact, I'd generally say only one kind of suit should be stored there, since it will probably be converted into a radial menu in the future.

File: Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml (hunk 1 above)
Comment (tom-leys) [reply Resources/Prototypes/Catalog/Fills/Lockers/engineer.yml:88]: Perhaps back into the locker?

**Pull request closed without merging**